EngineLayer/ProteinParsimony/ProteinParsimonyEngine.cs

```diff
@@ -27,7 +26,7 @@ public class ProteinParsimonyEngine : MetaMorpheusEngine
         public ProteinParsimonyEngine(List<PeptideSpectralMatch> allPsms, bool modPeptidesAreDifferent, CommonParameters commonParameters, List<string> nestedIds) : base(commonParameters, nestedIds)
```

Comment:
would be cool to add a comment that give the big picture overview of how parsimony is work in addition to the very nice comments that you have below explaining individual steps